pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -832,6 +889,8 @@ def set_distributed_mode(self, strategy: Optional[str] = None):
         if self.distributed_backend is None:
             if self.has_horovodrun():
                 self._set_horovod_backend()
+            elif self.num_hpus > 1 and not _use_cpu:
+                self.distributed_backend = _StrategyType.DDP
```

Comment:
Why not subclass DDP and create a new `HPUParallelStrategy` or `HPUStrategy` that supports single as well as parallel devices?